### PR TITLE
docs: command correction for gpasswd

### PR DESCRIPTION
**Pull request by ghost**
`gpasswd -d user wheel` needs sudo

### CODE REVIEW

**Review by EsseLowNitro — DISMISSED** [review]

**Comment by mintpilo** [comment]
For future proofing, wouldn't it be better to recommend run0 for when sudo is eventually removed? #296 It can currently be used on `latest`.

**Comment by ghost** [comment]
@mintpilo, good Idea!

**Review by EsseLowNitro — DISMISSED** [review]

File: docs/POSTINSTALL-README.md [hunk 1]
```diff
@@ -86,16 +86,16 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
 > [!CAUTION]
 > If you do these steps out of order, it is possible to end up without the ability to administrate your system. You will not be able to use the [traditional GRUB-based method](https://linuxconfig.org/recover-reset-forgotten-linux-root-password) of fixing mistakes like this, either, as this will leave your system in a broken state. However, simply rolling back to an older snapshot of your system, should resolve the problem.
 
-1. ```sudo adduser admin```
-2. ```sudo usermod -aG wheel admin```
-3. ```sudo passwd admin```
+1. ```run0 adduser admin```
```
Comment (RoyalOughtness) [inline docs/POSTINSTALL-README.md]: have you tested these with run0? It's not a drop-in sudo replacement so we need to verify that it works properly for this case

Comment (mintpilo) [reply docs/POSTINSTALL-README.md]: Update: I have tested this now, I've discussed the details in Discord, but in short: Every command works with `run0`, *except* for `passwd admin`.The command will only open a blank prompt, then immediately close. If you invoke the command with `sudo`, it works fine as usually expected. We could potentially explore alternatives, one suggestion @RoyalOughtness suggested is "could just `run0` for a root shell, run commands, then exit". (Or I assumed that was a suggestion 😅)

Comment (RoyalOughtness) [reply docs/POSTINSTALL-README.md]:
```
run0
adduser admin
usermod -aG wheel admin
passwd admin
exit
reboot
```

**Review by RoyalOughtness — DISMISSED** [review]

**Review by Rubiginosa — CHANGES_REQUESTED** [review]
For consistency with the rest of the doc I think single backticks would be preferred, unless there's a particular reason to use three.

**Comment by RoyalOughtness** [comment]
> For consistency with the rest of the doc I think single backticks would be preferred, unless there's a particular reason to use three.

@Rubiginosa the rest of the doc uses triple backticks for commands. Not sure what you mean.

**Comment by Rubiginosa** [comment]
> @Rubiginosa the rest of the doc uses triple backticks for commands. Not sure what you mean.

@RoyalOughtness Triple backticks are used in the doc for block commands across multiple lines. Here each line is in an ordered list, so it's a little odd to use three per line when one would work.

**Comment by EsseLowNitro** [comment]
The only distinction I could really find is that the docs are currently using triple backticks for single commands, even when they span multiple lines, for the sake of readability. Single (and, for some reason, double) backticks are being used when commands and other terms that need highlighting are within a phrase that itself doesn't.

**Review by EsseLowNitro — DISMISSED** [review]

**Comment by RoyalOughtness** [comment]
@Rubiginosa @EsseLowNitro updated, please re-review

**Review by EsseLowNitro — APPROVED** [review]

**Review by Rubiginosa — APPROVED** [review]

**Review by RoyalOughtness — APPROVED** [review]

**Pull request merged**